src/styles/main.scss

```diff
@@ -0,0 +1,70 @@
+@import '../styles/utils/variables';
+
+body {
+  margin: 0;
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  justify-content: center;
+  height: 100vh;
+
+  font-family: Arial, sans-serif;
+  font-size: $font-size;
+}
+
+.calendar {
+  margin-left: 14px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the indentation should be consistent throughout the project. It seems like you're using 2 spaces for indentation, but there's an extra space before the margin-left property. Please ensure that the indentation is consistent.